var/spack/repos/builtin/packages/jansson/package.py

```diff
@@ -14,6 +14,12 @@ class Jansson(CMakePackage):
     url      = "https://github.com/akheron/jansson/archive/v2.9.tar.gz"
     maintainers = ['ax3l']
 
+
```

Comment:
flake is complaining about the extra blank line here

Reply:
thanks. fixed in new revision